### DISCUSSION TITLE
增加对bi工具连接的支持

### DISCUSSION THREAD

**Discussion by HK-13** [vesoft-inc/nebula#4866] [Ideas]
目前很多图数据库基本仅提供了对图查询 UI的支持，比如nebula studio，neo4j browser产品等，但是图查询的统计分析需求却难有适配的工具，主流bi工具目前都缺乏对图数据库的支持，nebula未来是否可增加此特性。

neo4j好像有提供相关的bi connector(https://neo4j.com/blog/getting-started-with-the-neo4j-bi-connector/?ref=bi-connector)

**Comment by wey-gu** [comment]
cc @MuYiYong

**Comment by MuYiYong** [comment]
makes sense, and neo4j is implemented through a jdbc driver, it looks like it fits us as well.I will put it into the roadmap.
